index.html

```diff
@@ -15,146 +15,172 @@
     <link rel="icon" href="/images/favicon.ico" />
     <link rel="stylesheet" href="css/base.css" />
     <link rel="stylesheet" href="css/reset.css" />
-    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/index.css" />
     <title>판다마켓</title>
   </head>
   <body>
-    <header>
-      <nav class="nav">
-        <div class="nav-content">
-          <div class="logo">
-            <a href="/"
-              ><img src="images/pandamarket.png" width="153" height="51"
-            /></a>
+    <div class="content-wrapper">
+      <header>
+        <nav class="nav">
+          <div class="nav-content">
+            <div class="logo">
+              <a href="/">
+                <img src="images/pandamarket.png" width="153" height="51" />
+              </a>
+            </div>
+            <a href="/login.html" class="login"> 로그인 </a>
```

Comment:
❗️ 수정요청

지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 결과를 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
            <a href="/login.html" class="login">로그인</a>
```